crates/nu-explore/src/commands/help.rs

```diff
@@ -33,13 +33,14 @@ Launch Explore by piping data into it: {}
 
                    Move around:  Use the cursor keys
 Drill down into records+tables:  Press <Enter> to select a cell, move around with cursor keys, press <Enter> again
-            Go back/up a level:  Press <Esc>
+            Go back/up a level:  Press <Esc> or "q"
  Transpose (flip rows+columns):  Press "t"
  Expand (show all nested data):  Press "e"
           Open this help page :  Type ":help" then <Enter>
       Open an interactive REPL:  Type ":try" then <Enter>
-                Scroll up/down:  Use the "Page Up" and "Page Down" keys
-                  Exit Explore:  Type ":q" then <Enter>, or Ctrl+D. Alternately, press <Esc> until Explore exits
+                Scroll up:  Press "Page Up", Ctrl+B, or Alt+V
```

Comment:
This is a nit, but all the colons were lined up previously. Can we keep that alignment?

Reply:
Oh man, I'm so sorry, I'm usually quite good at noticing when colons are lined up, but I somehow missed this one. I've pushed a cleanup for this. Thanks!